build.gradle

```diff
@@ -4,64 +4,160 @@ buildscript {
             url "http://dl.bintray.com/jmoore/java-lib"
         }
     }
+    //noinspection GroovyAssignabilityCheck
     dependencies {
-        classpath 'com.mooregreatsoftware:gradle-defaults:1.0.3'
+        classpath 'com.mooregreatsoftware:gradle-defaults:1.0.6'
     }
 }
 
 plugins {
     id 'com.jfrog.bintray' version '1.4'
+    id "us.kirchmeier.capsule" version "1.0.2"
 }
 
 group = 'com.twcable.grabbit'
 description = 'Command line interface for interacting with the Grabbit Client'
 
 apply plugin: 'groovy'
 
-apply plugin: 'application'
-apply plugin: 'groovy'
-mainClassName = "com.twcable.grabbit.cli.Grabbit"
+//noinspection GroovyAssignabilityCheck
+configurations {
+    //noinspection GroovyAssignabilityCheck
+    checkerFrameworkAnnotatedJDK {
+        description = 'a copy of JDK classes with Checker Framework type qualifiers inserted'
```

Comment:
The verbiage is straight from the setup instructions from the Checker Framework. :wine_glass: 

Still getting a hang of ReviewNinja, but for "Minor:" items, should probably not use the "bang-fix" or "bang-resolve" statuses since they block the workflow from proceeding. !completed